src/nise/metadata/config.py

```diff
@@ -0,0 +1,26 @@
+config = {
+    'metadata_version': '1.0',
+    'module_name': 'nise',
+    'module_version': '1.0.1.dev0',
```

Comment:
Do you really want to start with version 1.0.1? I think you might want to make the first release something like 0.9.0 or something just in case there are breaking changes required.

Reply:
While I understand the reason that 0.9 signifies that we might break something, I don't like the idea of different versions for drivers and nimi-python. I like that nimi-python 1.0.1 supports nise 1.0.1.

 @marcoskirsch @texasaggie97  Thoughts?

Reply:
While we may think of nimi-python as a whole, 99.99% of customers will download each module separately, think of them as distinct things, and will appreciate different versions.

Reply:
Fixed!